tests/phpunit/tests/block-supports/spacing.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * @group block-supports
+ */
+class WP_Block_Supports_Spacing_Test extends WP_UnitTestCase {
+	private $sample_block_content = '<div class="wp-block-test-block">Test</div>';
+	private $test_gap_block_value = array();
+	private $test_gap_block_args  = array();
+
+	function setUp() {
+		parent::setUp();
```

Comment:
```suggestion
	function set_up() {
		parent::set_up();
```

During 5.9 cycle, the native PHPUnit fixture methods were changed to snake_case with the adoption of PHPUnit Polyfills (for running all PHP supported versions in Core).